snmp/metadata.csv

```diff
@@ -89,20 +89,23 @@ snmp.fgFwPol6ByteCount.rate,gauge,,packet,second,[Fortinet FortiGate] The number
 snmp.ifAdminStatus,gauge,,,,[Generic router] [F5 BIG-IP] [Cisco c3850] [Cisco Nexus] [Cisco ASA 5525] The desired state of the interface.,0,snmp,
 snmp.ifSpeed,gauge,,,,"[Generic router] An estimate of the interface's current bandwidth in bits per second, or the nominal bandwidth.",0,snmp,
 snmp.ifHighSpeed,gauge,,,,"[Generic router] An estimate of the interface's current bandwidth in units of 1,000,000 bits per second, or the nominal bandwidth.",0,snmp,
-snmp.ifBandwidthInUsage.rate,gauge,,percent,,"[Generic router] The percent rate of used received bandwidth.",0,snmp,
-snmp.ifBandwidthOutUsage.rate,gauge,,percent,,"[Generic router] The percent rate of used sent bandwidth.",0,snmp,
+snmp.ifBandwidthInUsage.rate,rate,,percent,,"[Generic router] The percent rate of used received bandwidth.",0,snmp,
+snmp.ifBandwidthOutUsage.rate,rate,,percent,,"[Generic router] The percent rate of used sent bandwidth.",0,snmp,
```

Comment:
```suggestion
snmp.ifBandwidthInUsage.rate,gauge,,percent,,"[Generic router] The percent rate of used received bandwidth.",0,snmp,
snmp.ifBandwidthOutUsage.rate,gauge,,percent,,"[Generic router] The percent rate of used sent bandwidth.",0,snmp,
```

I think this was correct. When using `aggregator.assert_metrics_using_metadata(get_metadata_metrics())` in integration tests, you should use the option `check_submission_type=True`.

Reply:
Should be good now, @AlexandreYang could you give another round of review please?